tiatoolbox/utils/misc.py

```diff
@@ -45,11 +47,14 @@ def split_path_name_ext(full_path):
     return input_path.parent.absolute(), input_path.name, input_path.suffixes
 
 
-def grab_files_from_dir(input_path, file_types=("*.jpg", "*.png", "*.tif")):
+def grab_files_from_dir(
+    input_path: PathLike,
+    file_types: Union[str, Tuple[str]] = ("*.jpg", "*.png", "*.tif"),
+) -> List[pathlib.Path]:
     """Grab file paths specified by file extensions.
 
     Args:
-        input_path (str or pathlib.Path):
+        input_path (PathLike):
```

Comment:
```suggestion
        input_path (str | PathLike):
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/620/commits/f729dc35407b7d347fd39bfb4a24dbe6856d4542